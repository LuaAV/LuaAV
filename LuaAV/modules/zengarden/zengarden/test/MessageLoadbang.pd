#N canvas 214 189 450 300 10;
#X obj 44 17 loadbang;
#X obj 44 117 print;
#X msg 72 66 34;
#X msg 114 66 hello;
#X msg 169 65 12 4 87;
#X connect 0 0 1 0;
#X connect 0 0 2 0;
#X connect 0 0 3 0;
#X connect 0 0 4 0;
#X connect 2 0 1 0;
#X connect 3 0 1 0;
#X connect 4 0 1 0;
