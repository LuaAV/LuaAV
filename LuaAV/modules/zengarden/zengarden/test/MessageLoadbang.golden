[@ 0.000ms] print: bang
[@ 0.000ms] print: 34
[@ 0.000ms] print: hello
[@ 0.000ms] print: 12 4 87
